setup.py

```diff
@@ -13,13 +13,15 @@
 allow to represent an arbitrary Python object.
 
 PyYAML is applicable for a broad range of tasks from complex
-configuration files to object serialization and persistence."""
+configuration files to object serialization and persistence.
+
+PyYAML-ft is a fork of PyYAML that adds support for free-threading."""
 AUTHOR = "Kirill Simonov"
 AUTHOR_EMAIL = 'xi@resolvent.net'
 LICENSE = "MIT"
 PLATFORMS = "Any"
 URL = "https://pyyaml.org/"
```

Comment:
We should probably update AUTHOR, AUTHOR_EMAIL and URL too. URL should point to our fork on github.